### PR TITLE
Java: regenerate Proto, Grpc and Gapic for DLP

### PR DESCRIPTION
**Pull request by neozwu**
regenerate proto, grpc and gapic for DLP

### CODE REVIEW

File: generated/java/gapic-google-cloud-dlp-v2beta1/src/main/java/com/google/cloud/dlp/v2beta1/DlpServiceClient.java [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+          CreateInspectOperationRequest, InspectOperationResult, InspectOperationMetadata>
+      createInspectOperationOperationCallable;
+  private final UnaryCallable<ListInspectFindingsRequest, ListInspectFindingsResponse>
+      listInspectFindingsCallable;
+  private final UnaryCallable<ListInfoTypesRequest, ListInfoTypesResponse> listInfoTypesCallable;
+  private final UnaryCallable<ListRootCategoriesRequest, ListRootCategoriesResponse>
+      listRootCategoriesCallable;
+
+  private static final PathTemplate RESULT_PATH_TEMPLATE =
+      PathTemplate.createWithoutUrlEncoding("inspect/results/{result}");
```
Comment (garrettjonesgoogle) [inline generated/java/gapic-google-cloud-dlp-v2beta1/src/main/java/com/google/cloud/dlp/v2beta1/DlpServiceClient.java]: We should configure resource name classes for the API

Comment (neozwu) [inline generated/java/gapic-google-cloud-dlp-v2beta1/src/main/java/com/google/cloud/dlp/v2beta1/DlpServiceClient.java]: pushed a new commit based on [this](https://github.com/googleapis/googleapis/pull/390). PTAL

**Comment by garrettjonesgoogle** [comment]
That's the only thing I noticed, let someone else approve too.

**Comment by neozwu** [comment]
closed this PR since a second [PR](https://github.com/googleapis/api-client-staging/pull/365) is created to trigger CircleCI.

**Pull request closed without merging**